cdap-app-fabric/src/main/java/io/cdap/cdap/gateway/handlers/AppLifecycleHttpHandler.java

```diff
@@ -232,18 +233,8 @@ public BodyConsumer createAppVersion(HttpRequest request, HttpResponder responde
                                        @PathParam("version-id") final String versionId)
```

Comment:
I believe we should mark this method as `@Deprecated`

Reply:
marked as `@Deprecated`